shell/src/main/java/alluxio/cli/fs/command/AbstractFileSystemCommand.java

```diff
@@ -28,4 +37,29 @@
   protected AbstractFileSystemCommand(FileSystem fs) {
     mFileSystem = fs;
   }
+
+  protected void runPath(AlluxioURI plainPath) throws AlluxioException, IOException{
+  }
+
+  protected void runWildCardCmd(AlluxioURI wildCardPath) throws IOException {
+    List<AlluxioURI> paths = FileSystemShellUtils.getAlluxioURIs(mFileSystem, wildCardPath);
+    if (paths.size() == 0) { // A unified sanity check on the paths
+      throw new IOException(wildCardPath + " does not exist.");
+    }
+    Collections.sort(paths, FileSystemShellUtils.createAlluxioURIComparator());
```

Comment:
try this: `paths.sort(Comparator.comparing(AlluxioURI::getPath))`

Reply:
done